instrumentation/concurrent_ruby/lib/opentelemetry/instrumentation/concurrent_ruby/instrumentation.rb

```diff
@@ -16,17 +16,21 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
         end
 
         present do
-          defined?(::Concurrent::Future)
+          defined?(::Concurrent::RubyExecutorService) || defined?(::Concurrent::JavaExecutorService)
         end
 
         private
 
         def require_dependencies
-          require_relative 'patches/future'
+          require_relative 'patches/executor_service'
         end
 
         def patch
-          ::Concurrent::Future.send(:include, Patches::Future)
+          if ::Concurrent.on_jruby?
+            ::Concurrent::JavaExecutorService.prepend Patches::ExecutorService
+          else
+            ::Concurrent::RubyExecutorService.prepend Patches::ExecutorService
+          end
```

Comment:
```suggestion
          ::Concurrent::RubyExecutorService.prepend Patches::ExecutorService          
          ::Concurrent::JavaExecutorService.prepend Patches::ExecutorService if ::Concurrent.on_jruby?
```

perhaps, should I change like above?